modules/gdscript/gdscript_function.cpp

```diff
@@ -128,8 +127,7 @@ Variant *GDScriptFunction::_get_variant(int p_address, GDScriptInstance *p_insta
 		} break;
 	}
 
-	ERR_EXPLAIN("Bad Code! (Addressing Mode)");
-	ERR_FAIL_V(NULL);
+	ERR_FAIL_V_MSG(NULL, "Bad Code! (Addressing Mode).");
```

Comment:
That's a bit cryptic, can we find something more explicit @bojidar-bg?
(Also, fix capitalization.)

Reply:
Looking at the code, maybe addressing mode -> unknown addressing mode at least?